From d4d8573dca1ba2398fa440318ac79d1a569ed5e6 Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 09:04:19 -0700
Subject: [PATCH 14/96] HBASE-7080 TestHFileOutputFormat.testColumnFamilyCompression failing due to UnsatisfiedLinkError

Reason: Test
Author: Jimmy Xiang
Ref: CDH-8628
(cherry picked from commit d4311dc1de4e324195cb8977bb7e6fe946cfce2b)
---
 .../hbase/mapreduce/TestHFileOutputFormat.java     |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/test/java/org/apache/hadoop/hbase/mapreduce/TestHFileOutputFormat.java b/src/test/java/org/apache/hadoop/hbase/mapreduce/TestHFileOutputFormat.java
index c21cd05..a06c91f 100644
--- a/src/test/java/org/apache/hadoop/hbase/mapreduce/TestHFileOutputFormat.java
+++ b/src/test/java/org/apache/hadoop/hbase/mapreduce/TestHFileOutputFormat.java
@@ -648,7 +648,7 @@ public class TestHFileOutputFormat  {
         Compression.Algorithm algo = Compression.getCompressionAlgorithmByName(algoName);
         algo.getCompressor();
         supportedAlgos.add(algo);
-      }catch (Exception e) {
+      } catch (Throwable t) {
         // this algo is not available
       }
     }
-- 
1.7.0.4

